teamB/cheonsongin/routes/postRouter.js

```diff
@@ -0,0 +1,14 @@
+const express = require('express');
+const postController = require('../controllers/postController');
+
+const router = express.Router();
+
+router.post('/upload', postController.upload);
+router.get('/viewallposts', postController.viewAllPosts);
+router.get('/user/:userid', postController.viewUserId);
+router.put('/update', postController.postUpdate);
+router.delete('/delete', postController.postDelete);
```

Comment:
위코드에서는 RESTful API 기반의 API를 작성합니다.

추후 세션을 통해 배울 내용이지만, RESTful API에서 엔드포인트를 구축할 때엔 동사가 아닌 명사로 작성합니다.

```suggestion
router.post('/posts', postController.upload);
router.get('/posts', postController.viewAllPosts);
router.get('/posts/:userid', postController.viewUserId);
router.put('/posts', postController.postUpdate);
router.delete('/posts', postController.postDelete);
```